requirements.txt

```diff
@@ -12,7 +12,7 @@ joblib
 psutil
 pyyaml
 liac-arff
-pandas<1.0
+pandas
```

Comment:
I don't know why this restriction was still here. Maybe an issue while merging?

Reply:
Me neither. As long as it works it's fine I guess.